container/container_windows.go

```diff
@@ -10,6 +10,10 @@ import (
 	containertypes "github.com/docker/docker/api/types/container"
 )
 
+const (
+	containerSecretMountPath = `C:\ProgramData\Docker\secrets`
```

Comment:
`ProgramData` really is the place for application data. But I guess as this is in the container itself, and it's not persistent, could it simply go somewhere unique under `TEMP`? Does it have to be in a well-known location from the containers perspective (guessing the answer to that is yes)?